.github/workflows/maven.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest ]
         java: ['1.8', '11']
-        wildfly-version: ['20.0.1.Final', '21.0.1.Final']
+        wildfly-version: ['21.0.1.Final', '22.0.1.Final']
```

Comment:
Thanks, I changed last commit, so MR adds WF 22.0.1 and 23.0.2. CI looks ok, just on job fails because of timeout (`Could not transfer artifact com.puppycrawl.tools:checkstyle:jar:8.19 from/to google-maven-central (https://maven-central.storage-download.googleapis.com/maven2/): GET request of: com/puppycrawl/tools/checkstyle/8.19/checkstyle-8.19.jar from google-maven-central failed: Connection reset`) which seems to be not related with this PR.

Reply:
Sorry, I forgot about this one.